core/src/test/scala/com/salesforce/op/stages/impl/preparators/SanityCheckerTest.scala

```diff
@@ -466,8 +466,13 @@ class SanityCheckerTest extends OpEstimatorSpec[OPVector, BinaryModel[RealNN, OP
       featuresToDrop, featuresWithNaNCorr)
   }
 
-  it should "not calculate correlations on hashed text features if asked not to (using transmogrify)" in {
-    val vectorized = Seq(textMap).transmogrify()
+  it should "not calculate correlations on hashed text features if asked not to (using vectorizer)" in {
+
+    val vectorized = textMap.vectorize(trackNulls = TransmogrifierDefaults.TrackNulls,
```

Comment:
Minor: You can just do `textMap.vectorize(cleanText = TransmogrifierDefaults.CleanText)`.

Reply:
fixed!